tests/RequestBuilder/SortingRequestBuilderTest.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Exception\LogicException;
+use Lmc\Matej\Http\RequestManager;
+use Lmc\Matej\Model\Command\Interaction;
+use Lmc\Matej\Model\Command\Sorting;
+use Lmc\Matej\Model\Command\UserMerge;
+use Lmc\Matej\Model\Request;
+use Lmc\Matej\Model\Response;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers \Lmc\Matej\RequestBuilder\SortingRequestBuilder
+ * @covers \Lmc\Matej\RequestBuilder\AbstractRequestBuilder
+ */
+class SortingRequestBuilderTest extends TestCase
+{
+    /** @test */
+    public function shouldBuildRequestWithCommands(): void
+    {
+        $builder = new SortingRequestBuilder('userId1', ['itemId1', 'itemId2']);
+
+        $interactionCommand = Interaction::detailView('userId1', 'itemId1');
+        $builder->addInteraction($interactionCommand);
+
+        $userMergeCommand = UserMerge::mergeFromSourceToTargetUser('sourceId1', 'targetId1');
+        $builder->addUserMerge($userMergeCommand);
+
+        $request = $builder->build();
+
+        $this->assertInstanceOf(Request::class, $request);
+        $this->assertSame(RequestMethodInterface::METHOD_POST, $request->getMethod());
+        $this->assertSame('/sorting', $request->getPath());
+
+        $requestData = $request->getData();
+        $this->assertCount(3, $requestData);
+        $this->assertContains($interactionCommand, $requestData);
+        $this->assertContains($userMergeCommand, $requestData);
+
+        // Check for equality of sorting command
+        $sortingCommand = Sorting::create('userId1', ['itemId1', 'itemId2']);
+        $this->assertContains($sortingCommand, $requestData, '', false, false);
+    }
+
+    /** @test */
+    public function shouldThrowExceptionWhenSendingCommandsWithoutRequestManager(): void
+    {
+        $builder = new SortingRequestBuilder('userId1', ['itemId1', 'itemId2']);
+
+        $this->expectException(LogicException::class);
+        $this->expectExceptionMessage('Instance of RequestManager must be set to request builder');
+        $builder->send();
+    }
+
+    /** @test */
+    public function shouldSendRequestViaRequestManager(): void
+    {
+        $requestManagerMock = $this->createMock(RequestManager::class);
+        $requestManagerMock->expects($this->once())
+            ->method('sendRequest')
+            ->with($this->isInstanceOf(Request::class))
+            ->willReturn(new Response(0, 0, 0, 0));
+
+        $builder = new SortingRequestBuilder('userId1', ['itemId1', 'itemId2']);
+        $builder->setRequestManager($requestManagerMock);
+
+        $builder->addInteraction(Interaction::detailView('userId1', 'itemId1'));
```

Comment:
Maybe this addInteraction and addUserMerge are not necessary for purposes of this unit test.